tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -834,22 +838,25 @@ cc_library(
         "//tensorflow/core:stream_executor_no_cuda",
         "@com_google_absl//absl/memory",
         "@llvm//:core",
-    ],
+    ], 
     alwayslink = True,  # Contains per-platform transfer manager registration
 )
 
 cc_library(
     name = "gpu_compiler",
     srcs = if_cuda_is_configured(["nvptx_compiler.cc"]) 
            + if_rocm_is_configured(["amdgpu_compiler.cc"]),
-    hdrs = if_cuda_is_configured(["nvptx_compiler.h"]) 
+    hdrs = [
+             "cudnn_batchnorm_rewriter.h",
```

Comment:
@sumesh13 could you explain rationale behind changes wrt `cudnn_batchnorm_rewriter`. doesn't seem quite right to me.